src/components/volunteer/VolunteerGroup.tsx

```diff
@@ -82,6 +88,18 @@ export const VolunteerGroup = ({
     isSuccess: isSuccessLeave
   } = useMutation(groupLeave);
 
+  useEffect(() => {
+    fetchGroupMembers();
+  }, []);
+
+  // added this request to add the IDs of the group's admins as param to the send message button
+  const fetchGroupMembers = async () => {
+    const { results } = await groupMembership({ id });
+    setGroupAdmins(
+      results?.filter(({ role }: any) => role === 'admin').map(({ user: { id } }: any) => id)
```

Comment:
which roles can exist? let's have a const enum object like for join policy, visibility and member status types.

Reply:
did it eade8fd